Source/WebCore/rendering/RenderLayer.h

```diff
@@ -1293,7 +1293,7 @@ class RenderLayer final : public CanMakeSingleThreadWeakPtr<RenderLayer>, public
                                  // we ended up painting this layer or any descendants (and therefore need to
                                  // blend).
     bool m_paintingInsideReflection : 1;  // A state bit tracking if we are painting inside a replica.
-    RepaintStatus m_repaintStatus : 2 { RepaintStatus::NeedsNormalRepaint };
```

Comment:
This will add 6 bits to each RenderLayer. We should manually implement the `operator==` instead ?

Reply:
Or wrap this change inside some `#ifdef`

Reply:
Actually just casting to the proper type is probably enough.

Reply:
Either way, it's late at night here, any chance someone at some can fix this regression or revert the original patch? Thanks.

Reply:
I'm not sure what GCC doesn't like about this, the existing code had the same comparison and it worked fine there.

Trying to reproduce the same issue in godbolt.org, I can't find a GCC version where it doesn't work :(